common/src/test/resources/index-mappings/test-mapping.json

```diff
@@ -1,6 +1,6 @@
 {
   "_meta": {
-    "schema_version": "1"
+    "schema_version": 1
```

Comment:
Usually, I see that JSON files in the repository are named using snake_case. Let's follow the same convention.